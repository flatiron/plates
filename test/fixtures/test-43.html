<div>
  <div id="include" data-ng-include="'path/to/partial'"></div>
  <div id="show" data-ng-show="svc.prop == 'val'"></div>
  <div data-ng-show="tabSvc.tab == 'locations'">Don't Touch Me</div>
</div>
