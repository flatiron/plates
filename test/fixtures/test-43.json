{
	  "fullname" : {
		  "firstname": "bob",
		  "lastname": "jones"
	  }
	}
