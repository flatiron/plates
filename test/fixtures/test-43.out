<div>
  <div id="include" data-ng-include="'path/to/partial'">Include</div>
  <div id="show" data-ng-show="svc.prop == 'val'">Show</div>
  <div data-ng-show="tabSvc.tab == 'locations'">Don't Touch Me</div>
</div>
